docs/guides/represent-quantum-computers.ipynb

```diff
@@ -86,10 +86,7 @@
    "id": "b133229c-0146-4feb-a366-548d175b858e",
    "metadata": {},
    "source": [
-    "<span id=\"default-config\"></span>\n",
-    "## Default configuration\n",
-    "\n",
-    "This example uses the default `optimization_level=1` to transpile to the `backend`'s `target`, which provides all the information needed to convert the circuit to one that will run on the system.  (Note that if a `target` is specified in the `transpile` function, this will override the target from the `backend`.)"
+    "This example uses default settings to transpile to the `backend`'s `target`, which provides all the information needed to convert the circuit to one that will run on the system. "
```

Comment:
```suggestion
    "This example uses default settings to transpile to the `backend`'s `target`, which provides all the information needed to convert the circuit to one that will run on the QPU. "
```

Reply:
I didn't change "system"s because I think this was already changed in the terminology PR and I didn't want to cause conflicts.